torch/_dynamo/trace_rules.py

```diff
@@ -3450,6 +3450,7 @@ def _module_dir(m: types.ModuleType) -> Optional[str]:
     "torch.utils._foreach_utils",
     "torch.utils._python_dispatch",
     "torch.utils._pytree",
+    "torch.utils.checkpoint",
```

Comment:
noob question: why do we also need to do this as part of the fix?

Reply:
We don't need to ... just removed